protos/sigstore_common.proto

```diff
@@ -71,6 +71,8 @@ message HashOutput {
 // MessageSignature stores the computed signature over a message.
 message MessageSignature {
         // Message digest can be used to identify the artifact.
+        // Clients MUST NOT attempt to use this digest to verify the associated
+        // signature; it is intended solely for identification.
```

Comment:
Can we drop the required field behavior annotation?

Reply:
My bad, I thought I did! Dropping.